src/Buttons/Button.tsx

```diff
@@ -24,17 +24,13 @@ export interface ButtonProps {
 
 const StyledButton = styled.button<ButtonProps>`
   background: ${({ theme, variant }): string => (theme.color.background[variant].medium)};
-  border-color: ${({ theme, variant }): string => (theme.color.background[variant].medium)};
-  border-style: solid;
-  border-width: 1px;
```

Comment:
We need to add `border: none` to remove the default browser button border:

![2020-02-25_160137](https://user-images.githubusercontent.com/6124793/75287373-3b787280-57e8-11ea-832c-b328689eb21f.png)

Reply:
Ah, I must be going blind! Fixed through 20d8b7d